### PR TITLE
New extensions : Pageloader & Ribbon

### PR DESCRIPTION
**Pull request by Wikiki**
Add links to the following extensions:

- Pageloader

- Ribbon

### CODE REVIEW

**Comment by jgthms** [comment]
Can you add an image for each in `/images/extensions`?

**Comment by Wikiki** [comment]
Done.

but sorry I don't know how to update the existing PR with my new commit.

**Comment by kevinvdburgt** [comment]
Wow, you have made some nice extensions! Awesome, I like them! ❤️

**Comment by Wikiki** [comment]
@kevinvdburgt Thank you :-)

File: docs/extensions.html [hunk 1]
```diff
@@ -2,6 +2,16 @@
 layout: default
 route: extensions
 extensions:
+- name: bulma-pageloader
+  url: https://github.com/Wikiki/bulma-pageloader
+  slug: bulma-pageloader
+  width: 455
+  height: 133
```
Comment (jgthms) [inline docs/extensions.html]: The `height` and `width` should match the image dimensions.

**Comment by Wikiki** [comment]
Close it because I open new one (issue on my side with the PR)

**Pull request closed without merging**